src/commonMain/kotlin/com.monta.changelog/GenerateChangeLogCommand.kt

```diff
@@ -88,7 +93,8 @@ class GenerateChangeLogCommand : CliktCommand() {
                 tagSorter = versionMode.sorter,
                 githubRelease = githubRelease,
                 githubToken = githubToken,
-                tagPattern = tagPattern
+                tagPattern = tagPattern,
+                pathExcludePattern = pathExcludePattern
```

Comment:
nitpick (non-blocking): consider using trailing commas everywhere to minimize diff noise.

Reply:
@dirkhas this is probably something to standardize everywhere if that's the case, because right now it's disabled in some `.editorconfig`s and not set at all in others 

```.editorconfig
[*.{kt,kts}]
indent_style = space
indent_size = 4
ij_any_if_brace_force = always

[*.{kt,kts}]
ktlint_disabled_rules = no-wildcard-imports
ij_kotlin_allow_trailing_comma_on_call_site = false
ij_kotlin_allow_trailing_comma = false
```

This is an example of what one of them looks...